pom.xml

```diff
@@ -113,11 +113,14 @@
             <version>${kotlin.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
only what we need for the actual PR / change not open-cypher-dsl

Reply:
I will remove as soon we get an release of `open-cypher-dsl`